contrib/dockerd-rootless.sh

```diff
@@ -128,5 +130,7 @@ else
 		mount --rbind ${realpath_etc_ssl} /etc/ssl
 	fi
 
-	exec dockerd $@
+	# shellcheck disable=SC2068
+	# shellcheck disable=SC2086
+	exec $dockerd $@
```

Comment:
```suggestion
	# shellcheck disable=SC2086
	exec $dockerd "$@"
```

Reply:
I had to fix some shell formatting issues, so I've applied this one as well. :ok_hand: